lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -82,7 +82,9 @@ class CategoryRenderer {
     const titleEl = this.dom.find('.lh-audit__title', auditEl);
     titleEl.appendChild(this.dom.convertMarkdownCodeSnippets(audit.result.title));
     this.dom.find('.lh-audit__description', auditEl)
-        .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
+        .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description, {
```

Comment:
if we including the `rating`, do we need to do this in `PerformanceCategoryRenderer._renderMetric()` as well? And is there something structural we can do make it basically the default that this is needed whenever we add a new category renderer and it tries to make links?

Reply:
No longer applicable now that rating is removed.